src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -24,6 +24,7 @@
         "twig/twig": "~1.28|~2.0"
     },
     "require-dev": {
+        "symfony/console": "~3.3",
```

Comment:
It was already the case for the templating component in 3.3. AFAIK we allow this kind of changes (requiring to install a new dependency).

I'm not against adding Console as a hard dependency of FrameworkBundle, but I thought I understood one of our goal for 3.3/Flex was to have less components installed by default when requiring framework-bundle. This change go in the other direction.